openedx/features/wikimedia_features/meta_translations/models.py

```diff
@@ -0,0 +1,55 @@
+
+"""
+Meta Translations Models
+"""
+from django.db import models
+from common.djangoapps.student.models import LanguageField
+from opaque_keys.edx.django.models import CourseKeyField
+
+APP_LABEL = 'meta_translations'
+
+
+class CourseBlock(models.Model):
+    """
+    Store block_id(s) of base course blocks and translated reruns course blocks
+    """
+    block_id = models.CharField(max_length=255, unique=True)
```

Comment:
do we need to add indexing on `block_id`?

Reply:
reference: https://docs.djangoproject.com/en/4.0/ref/models/fields/#unique

> Note that when unique is True, you don’t need to specify [db_index](https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.Field.db_index), because unique implies the creation of an index.